webserver/webserver/src/main/java/io/helidon/webserver/ListenerConfigBlueprint.java

```diff
@@ -400,4 +400,11 @@ default void configureSocket(ServerSocket socket) {
         }
     }
 
+    /**
+     * Configuration for this listener's error handling.
+     *
+     * @return optional error handling
+     */
+    @Option.Configured
+    Optional<ErrorHandling> errorHandling();
```

Comment:
This could probably return `ErrorHandling` instead of `Optional<ErrorHandling>` using `@io.helidon.builder.api.Option.DefaultMethod("create")`, as both options have sensible defaults.
This would make usage a bit nicer.